src/mlpack/core/optimizers/rmsprop/rmsprop.hpp

```diff
@@ -2,8 +2,9 @@
  * @file rmsprop.hpp
  * @author Ryan Curtin
  * @author Marcus Edel
+ * @author Vivek Pal
  *
- * RMSprop optimizer. RmsProp is an optimizer that utilizes the magnitude of
+ * RMSprop optimizer. RMSprop is an optimizer that utilizes the magnitude of
```

Comment:
Yes, please, people probably heard of RMSProp or read about it in some paper and might be confused if we use a different name.